src/AppBundle/Model/EventWiki.php

```diff
@@ -159,7 +158,9 @@ public function getChildWikis()
         $family = $this->getFamilyName();
 
         return $this->event->getWikis()->filter(function (EventWiki $wiki) use ($family) {
-            return substr($this->domain, 2) === $family && $wiki->getDomain() !== $this->domain;
+            return $wiki->isChildWiki() &&
+                substr($this->domain, 2) === $family &&
```

Comment:
You should be careful with this; we have Wikipedias whose language code is more than 2 characters, for example,  ksh.wikipedia and the more outrageously-long one, http://zh-min-nan.wikipedia.org

(see more here https://en.wikipedia.org/wiki/List_of_Wikipedias)

Maybe split the string by periods, and check the first piece?

Reply:
There's actually a `EventWiki::getFamilyName()` method that I should be using here. Fixed!